tests/infra/network.py

```diff
@@ -507,6 +513,7 @@ def recover(self, args):
         Recovers a CCF network previously started in recovery mode.
         :param args: command line arguments to configure the CCF nodes.
         """
+        self.consortium.activate(self.find_random_node())
```

Comment:
Seems like an unrelated change to me?

Reply:
Good spot. Yes, this slipped through. I'd like to leave this in though as we didn't test this before: "ack"tivating members during the recovery process.